### DISCUSSION TITLE
Token lifetime and refresh flow

### DISCUSSION THREAD

**Discussion by oliver-steinbrecher** [Digitaler-Impfnachweis/certification-apis#15] [Q&A] [answered]
For PVS integrators the [SMC-B flow](https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/SMCB-Authentication.md) issues an access and refresh token with different lifetimes.  Once these tokes are issued the integrator can perform a refresh token flow to receive a new token. A new login is required when the session is expired.

**Comment by fue-medatixx** [comment, marked as answer]
What are the detailed steps for refresh token flow?

**Reply by oliver-steinbrecher** [reply]
We're leveraging a standard OIDC endpoint  hence the refresh token flow is specified by the OIDC spec : https://openid.net/specs/openid-connect-core-1_0.html#RefreshTokens
